Limitations.md

```diff
@@ -221,11 +221,15 @@ See more documentation at
 
 Privileged support in Kata is essentially different from `runc` containers.
 Kata does support `docker run --privileged` command, but in this case full access
-to the guest VM is provided instead of the host.
+to the guest VM is provided in addition to some host access.
+
 The container runs with elevated capabilities within the guest and is granted 
 access to guest devices instead of the host devices.
 This is also true with using `securityContext privileged=true` with Kubernetes.
 
+The container may also be granted full access to all host devices
```

Comment:
Need to check with @amshinde if this is fully true, that it gets access to **all** host devices or not...

Reply:
It does in fact mount all host devices into the VM (as long as they _can_ be mounted).